src/components/Common/MarkdownPreview.tsx

```
@@ -0,0 +1,99 @@
+        const username = ele.getAttribute("data-username");
+        if (username) setHoveredUser(username);
+      });
+      ele.addEventListener("mouseleave", () => {
+        setHoveredUser(null);
+      });
+    });
+  }, [sanitizedHtml]);
+
+  return (
+    <div className="relative prose text-sm prose-p:m-0">
+      <div dangerouslySetInnerHTML={{ __html: sanitizedHtml }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)